docs/contributors/code/release.md

```diff
@@ -1,149 +1,263 @@
+Once the release changelog is available in the draft, take some time to read the notes and edit them to make sure they are easy to read and accurate. Don't rush this part. It's important to make sure the release notes are as organized as possible, but you don't have to finish them all at once. You can save the draft and come back to it later.
 
-You can find some more tips on writing the release notes and post in the section below.
+If you're worried that people won't be able to access the release candidate version until you publish the release, you can share the release artifact with the [#core-editor](https://wordpress.slack.com/messages/C02QB2JS7) Slack channel. This will give people access to the release candidate version while you finish curating the changelog.
 
-#### Creating Release Candidate Patches
+Here are some additional tips for preparing clear and concise changelogs:
 
-At any point after the release candidate has been published but before the final stable release, some bugs related to this release might be fixed and committed to `trunk`. These fixes won't automatically be incorporated into the final stable release, including them is a manual process.
+* Move all entries under the `Various` section to a more appropriate section.
+* Fix spelling errors or clarify wording. Phrasing should be easy to understand where the intended audience is those who use the plugin or are keeping up with ongoing development.
+* Create new groupings as applicable, and move pull requests between.
+* When multiple PRs relate to the same task (such as a follow-up pull request), try to combine them into a single entry. Good examples for this are PRs around removing Lodash for performance purposes, replacement of Puppeteer E2D tests with Playwright or efforts to convert public components to TypeScript.
+* If subtasks of a related set of PRs are substantial, consider organizing as entries in a nested list.
+* Remove PRs that revert other PRs in the same release if the net change in code is zero. 
+* Remove all PRs that only update the mobile app. The only exception to this rule is if the mobile app pull request also updates functionality for the web.
+* If a subheader only has one PR listed, remove the subheader and move the PR to the next matching subheader with more than one item listed.
 
-There are a couple of ways a release coordinator might be made aware of these bugs:
+#### Creating release candidate patches (cherry-picking)
 
-- Contributors may add the `Backport to Gutenberg RC` label to a closed PR. [Do a search for any of these PRs](https://github.com/WordPress/gutenberg/pulls?q=is%3Apr+label%3A%22Backport+to+Gutenberg+RC%22+is%3Aclosed) before publishing the final release.
-- You may receive a direct message or a ping in the #core-editor channel in slack notifying you of PRs that need to be included in the release candidate. Even when this is the case, the `Backport to Gutenberg RC` should be added to the PR.
+QUICK REFERENCE!!!!!!!!
```

Comment:
Is this intended? Looks like a placeholder 😅

Reply:
🤦‍♂️ Yup, I was thinking about doing "Quick references" in each section that provide the top-level bullet points for those more familiar with the process who just need a reminder of the steps. Precisely what you highlighted 👇

> In that regard, what do you think about adding a TL;DR; in a follow-up? I worry folks get discouraged from handling releases by seeing a long documentation and getting the wrong impression. 🤔

I was thinking about this being in a follow-up, but I might as well get it done in this PR. I'll work on adding that today.

Reply:
This has been fixed, and I added a few "Quick reference" callouts to various sections. We may want to add more in the future, but I think this, plus the addition of the TOC, makes this doc much easier to work through.

Reply:
Fantastic! Let's merge!